modules/drivers/snowflake/test/metabase/test/data/snowflake.clj

```diff
@@ -144,12 +144,26 @@
         (delete-old-datasets!)
         (reset! deleted-old-datasets? true)))))
 
+(defn- set-current-user-timezone!
```

Comment:
Effect of this was and is, that user of the instance tests are running on has their timezone set permanently to UTC. Even after running from this branch. I was checking for some negative impact this could have after running it but to this point I've found none. (No CI drivers failures related to this I mean.)